analyzer/src/main/java/org/jboss/gm/analyzer/alignment/DAAlignmentService.java

```diff
@@ -36,13 +36,13 @@
     public DAAlignmentService(Configuration configuration) {
         final String endpointUrl = configuration.daEndpoint();
 
-        if (endpointUrl == null) {
+        dependencySource = configuration.dependencyConfiguration();
+
+        if (endpointUrl == null && (dependencySource != NONE)) {
             throw new ManipulationUncheckedException(
-                    String.format("'%s' must be configured in order for model to work", Configuration.DA));
+                    String.format("'%s' must be configured in order for dependency scanning to work", Configuration.DA));
         }
 
-        dependencySource = configuration.dependencyConfiguration();
-
```

Comment:
Refactored the order so if you specify `dependencySource=NONE` you can run without specifying a restURL.

Reply:
:+1: